tests/parallels/open-affine.spec.ts

```diff
@@ -26,33 +26,6 @@ test('Open last workspace when back to affine', async ({ page }) => {
   expect(currentWorkspaceName).toEqual('New Workspace 2');
 });
 
-test('Open affine in first time after updated', async ({ page }) => {
-  await openHomePage(page);
-  const changeLogItem = page.locator('[data-testid=change-log]');
-  await expect(changeLogItem).toBeVisible();
-  const closeButton = page.locator('[data-testid=change-log-close-button]');
-  await closeButton.click();
-  await expect(changeLogItem).not.toBeVisible();
-  await page.goto('http://localhost:8080');
-  const currentChangeLogItem = page.locator('[data-testid=change-log]');
```

Comment:
Im not sure if it is good to just remove these test cases.

Reply:
Change Log component is removed and will be integrated into the app updater button later